src/components/ChatEntry.js

```diff
@@ -2,21 +2,45 @@ import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
+const filledHeart = '🐣'; //Design choice: using eggs and chicks!
+const emptyHeart = '🥚';
+const user = 'Vladimir'; //You can choose between Vladimir and Estragon to switch sides
+
 const ChatEntry = (props) => {
+  const heartType = props.liked ? filledHeart : emptyHeart;
+  const buttonClass =
+    props.sender === user ? 'chat-entry local' : 'chat-entry remote';
+
+  const timeSince = (dateAsString) => {
+    const originalTime = new Date(dateAsString);
+    const currentTime = Date.now();
+    const diffTime = Math.abs(originalTime - currentTime);
+    const diffYears = Math.floor(diffTime / (1000 * 60 * 60 * 24 * 365));
+    return `${diffYears.toString()} years ago`;
+  };
```

Comment:
Oh no did you make your own function for this?? Good for you! There is one in the `TimeStamp` component for you to use, but I'm impressed you researched how to make your own!